### PR TITLE
kernel: bump 5.15 to 5.15.106

### PR DESCRIPTION
**Pull request by graysky2**
Removed upstreamed:
	generic/735-net-ethernet-mtk_eth_soc-fix-flow_offload-related-re.patch[1]
	pending-5.15/350-mips-bmips-BCM6358-disable-RAC-flush-for-TP1.patch[2]

All other patches automatically rebased.

1. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v5.15.106&id=76f09582a191dcf11118fd4bdbf50f538c90fa8d

2. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/arch/mips/bmips?h=v5.15.106&id=65b723644294f1d79770704162c0e8d1f700b6f1

Build system: x86_64
Build-tested: bcm2711/RPi4B, ramips/tplink_archer-a6-v3, filogic/xiaomi_redmi-router-ax6000-ubootmod, ipq806x/R7800
Run-tested: bcm2711/RPi4B, ramips/tplink_archer-a6-v3, filogic/xiaomi_redmi-router-ax6000-ubootmod, ipq806x/R7800

### CODE REVIEW

**Review by CharlesMengCA — COMMENTED** [review]
Tested working on mediatek/mt7622/linksys_e8450/ubi-squashfs-sysupgrade

File: target/linux/generic/pending-5.15/350-mips-bmips-BCM6358-disable-RAC-flush-for-TP1.patch [hunk 1]
```diff
@@ -135,39 +135,39 @@ Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>
...
-@@ -74,6 +76,9 @@ void arch_sync_dma_for_cpu_all(void)
- 	    boot_cpu_type() != CPU_BMIPS4380)
+@@ -77,6 +79,9 @@ void arch_sync_dma_for_cpu_all(void)
  		return;
  
-+	if (unlikely(bmips_rac_flush_disable))
+ 	if (unlikely(bmips_rac_flush_disable))
 +		return;
 +
++	if (unlikely(bmips_rac_flush_disable))
```
Comment (DragonBluep) [inline target/linux/generic/pending-5.15/350-mips-bmips-BCM6358-disable-RAC-flush-for-TP1.patch:153]: This patch can be removed.
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/arch/mips/bmips?h=v5.15.106&id=65b723644294f1d79770704162c0e8d1f700b6f1

Comment (graysky2) [reply target/linux/generic/pending-5.15/350-mips-bmips-BCM6358-disable-RAC-flush-for-TP1.patch]: You are correct, thanks.

File: target/linux/bmips/patches-5.15/200-mips-bmips-automatically-detect-CPU-frequency.patch [hunk 2]
```diff
@@ -58,17 +58,19 @@ Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>
...
  
+@@ -40,6 +74,11 @@ extern bool bmips_rac_flush_disable;
+ 
  static const unsigned long kbase = VMLINUX_LOAD_ADDRESS & 0xfff00000;
  
 +struct bmips_cpufreq {
 +	const char		*compatible;
 +	u32			(*cpu_freq)(void);
-+};
++}
```
Comment (hauke) [inline target/linux/bmips/patches-5.15/200-mips-bmips-automatically-detect-CPU-frequency.patch:68]: This change looks wrong to me. 
Why did your remove the `;`?

Comment (graysky2) [reply target/linux/bmips/patches-5.15/200-mips-bmips-automatically-detect-CPU-frequency.patch]: I see now that removing the other patch mentioned by @dragonbluep and refreshing did not need modifications to this patch at all.

**Pull request merged**